src/kirby/components/card/card.component.scss

```diff
@@ -5,14 +5,14 @@ $max-card-height: 500px;
 $min-card-width: 304px;
 
 :host{ 
-  --kirby-card-header-background-color:  #{get-color('kirby-white')};
+  --kirby-card-header-background-color:  #{get-color('kirby-contrast-light')};
```

Comment:
```suggestion
  --kirby-card-header-background-color:  #{get-color('kirby-primary')};
```

Reply:
I've added the background to the header - but I left it with contrast-light - otherwise the header becomes green, I don't think that is the intention?